docs/source/contribution/development_for_databricks.md

```diff
@@ -0,0 +1,73 @@
+# Developing Kedro for Databricks
+
+[Databricks](https://www.databricks.com/) is a cloud-based platform for data engineering and data science that many Kedro users deploy their Kedro projects to. Extending and improving the Kedro experience on Databricks is important to a large part of Kedro's user base.
+
+## Introduction
+
+This page describes how to efficiently develop features and fixes for Kedro on Databricks. Using this guide, you will be able to quickly test your locally modified version of Kedro on Databricks as part of a build-and-test development cycle.
+
+## Prerequisites
```

Comment:
You've not made any mention of different OS -- are the instructions at all different depending on whether you're on Windows or MacOS? For example, installing GNUMake or Git?

Reply:
All requirements are available on all major operating systems.